challenge-manager/edge-tracker/tracker.go

```diff
@@ -189,6 +189,7 @@ func (et *Tracker) Spawn(ctx context.Context) {
 				return
 			}
 			if err := et.Act(ctx); err != nil {
+				fields["err"] = err
 				srvlog.Error("Could not act with edge tracker", err)
```

Comment:
I have no context here, but is this intentional? `fields` seems to not be used in this log statement

Reply:
Thanks!